src/tablib/formats/_ods.py

```diff
@@ -7,7 +7,11 @@
 from odf import opendocument, style, table, text
 
 bold = style.Style(name="bold", family="paragraph")
-bold.addElement(style.TextProperties(fontweight="bold", fontweightasian="bold", fontweightcomplex="bold"))
+bold.addElement(style.TextProperties(
+    fontweight="bold",
+    fontweightasian="bold",
+    fontweightcomplex="bold"
```

Comment:
Trailing comma, so if we add another, only one line is in the diff:
```suggestion
    fontweightcomplex="bold",
```

Reply:
Fixed